website/public/js/services/userServices.js

```diff
@@ -259,12 +266,19 @@ angular.module('habitrpg')
         },
 
         unlock: function (data) {
-          $window.habitrpgShared.ops['unlock'](user, data);
           callOpsFunctionAndRequest('unlock', 'unlock', "POST", '', data);
         },
 
         set: function(updates) {
           setUser(updates);
+          $http({
+            method: "PUT",
+            url: 'api/v3/user',
+            data: updates,
+          })
+          .then(function (response) {
+            sync();
```

Comment:
Eh, I looked into this, and I believe I needed to sync for some reason, but can't remember. I'll try it again.